### PR TITLE
Added stream_decode parameter to execute_view_dataframe

### PR DESCRIPTION
**Pull request by raeldor**
Pulling large data sets resulted in extremely large memory usage because response.json() creates a dict of the entire dataset which blows the memory up to about 10x the usage of the JSON alone.  Added some (rudimentary) code to allow decoding of the JSON direct into CSV by streaming using ijson.

### CODE REVIEW

**Comment by raeldor** [comment]
Just FYI, I may not have started using the very latest version as a base.

File: TM1py/Services/CellService.py [hunk 1]
```diff
@@ -1840,11 +2091,19 @@ def extract_cellset_dataframe(
...
+                                            skip_rule_derived_cells=skip_rule_derived_cells,
+                                            skip_consolidated_cells=skip_consolidated_cells, value_separator='~',
+                                            sandbox_name=sandbox_name, include_attributes=include_attributes,
+                                            **kwargs)
+        else:
+            raw_csv = self.extract_cellset_csv(cellset_id=cellset_id, top=top, skip=skip, skip_zeros=skip_zeros,
+                                            skip_rule_derived_cells=skip_rule_derived_cells,
+                                            skip_consolidated_cells=skip_consolidated_cells, value_separator='~',
+                                            sandbox_name=sandbox_name, include_attributes=include_attributes,
+                                            **kwargs)
```
Comment (MariusWirtz) [inline TM1py/Services/CellService.py:2105]: the same code is executed in both conditions or am I missing something?

**Comment by raeldor** [comment]
I can't see your comment that I received by email on this page.  Weird.  No, not the same code, they're calling two different functions.

**Comment by MariusWirtz** [comment]
I deleted my comment immediately after I wrote it. I missed something.

I'm reviewing the code in more depth now. I will create a new PR with your commits based on the current master branch of TM1py

**Comment by MariusWirtz** [comment]
Closed in favor of #612

**Pull request closed without merging**